src/main.js

```diff
@@ -17,6 +18,7 @@ require('update-electron-app')();
 
 const isDevEnv = process.env.NODE_ENV === 'development';
 if (isDevEnv) app.setPath('userData', app.getPath('userData') + '_Dev');
+if (isDevEnv) debug({ isEnabled: true, showDevTools: false }); // Press F12 to open DevTools
```

Comment:
`electron-debug` says:

> Only runs when in development, unless overridden by the isEnabled option. So no need to guard it for production.
> (from https://www.npmjs.com/package/electron-debug#api)

So, it will work with this:
```suggestion
debug({ showDevTools: false }); // Press F12 to open DevTools
```

Reply:
Thanks. Fixed it.